src/scraper/scraper.cpp

```diff
@@ -1162,11 +1181,11 @@ bool ScraperDirectoryAndConfigSanity()
                 }
 
                 // End LOCK(cs_StructScraperFileManifest)
-                if (fDebug3) _log(logattribute::INFO, "ENDLOCK", "cs_StructScraperFileManifest");
+                if (fDebug3) _log(logattribute::INFO, "ENDLOCK", "align directory with manifest file: cs_StructScraperFileManifest");
             }
 
             // If network policy is set to filter on whitelisted teams, then load team ID map from file. This will prevent the heavyweight
-            // team file downloads for projects whose team ID's have already been found and stored.
+            // team file downloads for projects whose team ID's have already been found and stored, unless explorer mode (fExplorer) is true.
```

Comment:
Maybe I'm misreading--I see this comment, but there's no check for `fExplorer` below to skip loading the team ID map from disk.

Reply:
Exactly

Reply:
But if you aren't in explorer mode, and you have already "cached" the TeamID's then no need to download again.

This was a bit tricky... :)